demo/sections/layouts/LayoutGrid.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <ComponentPage title="Grid Layout" :demos="[{ title: 'Well Layout' }]">
+    <template #well-layout>
+      <p-layout-well-toolbar class="p-background">
+        <template #header-leading>
+          <div class="h-full">
+            Header Leading
+          </div>
+        </template>
+        <template #header-trailing>
+          <div class="h-full">
+            Header Trailing
+          </div>
+        </template>
+        <template #toolbar-leading>
+          <div class="h-full">
+            Toolbar Leading
+          </div>
+        </template>
+        <template #toolbar-trailing>
+          <div class="h-full">
+            Toolbar Trailing
+          </div>
+        </template>
+        <div class="p-background h-full">
+          <div class="p-background layout-well-section__box" />
+          <div class="p-background layout-well-section__box" />
+          <div class="p-background layout-well-section__box" />
+          <div class="p-background layout-well-section__box" />
+          <div class="p-background layout-well-section__box" />
+        </div>
+        <template #left-rail>
+          <div class="h-full">
+            Left Rail
+          </div>
+        </template>
+        <template #right-rail>
+          <div class="h-full">
+            Right Rail
+          </div>
+        </template>
```

Comment:
These aren't showing up in the demo because your slots are named `leftRail` and `rightRail`, respectively